core/src/main/java/org/apache/carbondata/core/util/DeleteLoadFolders.java

```diff
@@ -227,34 +224,9 @@ public static boolean deleteLoadFoldersFromFileSystem(
     if (details != null && details.length != 0) {
       for (LoadMetadataDetails oneLoad : details) {
         if (checkIfLoadCanBeDeleted(oneLoad, isForceDelete)) {
-          ICarbonLock segmentLock = CarbonLockFactory.getCarbonLockObj(absoluteTableIdentifier,
```

Comment:
after this PR, method deleteLoadFoldersFromFileSystem will be invoked by clean files only, right?
if yes, why we remove this code?

Reply:
Actually, this part of the modifications comes from #3971 contributed by @Indhumathi27 .  It removes the condition of insert_in_progress and lock acquirement, I think it may delete the segments which are in progress. So maybe I can revoke this modification.